.github/workflows/rust.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         run: cargo xtask build-client
 
   tests:
-    runs-on: ubuntu-24.04
```

Comment:
Given that the tests are platform independent, I don't think it matters what version it's ran on so why not just keep it as 24